proto/stride/stakeibc/host_zone.proto

```diff
@@ -124,6 +124,9 @@ message HostZone {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable) = false
   ];
+  // The max number of messages that can be sent in a delegation
+  // or undelegation ICA tx
+  uint64 max_messages_per_ica_tx = 36;
```

Comment:
Why would this number vary per hostZone?  It totally makes sense on the tx proto but what factors would cause it to be different per hostZone and if there are such factors how will we experimentally figure it out for each one?  Can it be automatically measured and if so can we set this param empirically instead of through tx setting up hostZones

Reply:
it's based on the gas limit on each host zone - which is up to the chain's governance so it can change at any time. And yeah initially, I think we'll have to set it based on experimentation

Reply:
Hmm interesting, maybe the two values have to change together?  If governance changes the gas value but this setting remains too high will that cause tx with too many messages to be attempted and they will fail because of this?  If so then the governance to change the gas fees should update this value at the same time always

Reply:
To clarify - it's the governance on the host zone that decides the gas limit. So the hub could put up a proposal to reduce the gas limit, and, as you said, it would cause these ICAs to start failing. And then stride would have to put up a prop to change the max messages to take the new limit into account